internal/cmd/group_flag.go

```diff
@@ -229,7 +232,7 @@ func dumpSQLiteDatabase(database string, dump *os.File) error {
 	return nil
 }
 
-func handleCSVFile(client *turso.Client, file, csvTableName string) (*turso.DBSeed, error) {
+func handleCSVFile(client *turso.Client, file, csvTableName string, csvSeparator string) (*turso.DBSeed, error) {
```

Comment:
⛏️: you're inside the context of the `handleCSVFile` you can call it just `separator` with no loss on semantics. 

```suggestion
func handleCSVFile(client *turso.Client, file, csvTableName string, separator string) (*turso.DBSeed, error) {
```